shared/middleware/helpers/table.rb

```diff
@@ -74,7 +74,7 @@ def update(id, value, ip_address)
 
   def to_a()
     items.map do |row|
-      JSON.load(row[:value]).merge(id: row[:row_id])
+      JSON.load(row[:value]).merge('id' => row[:row_id])
```

Comment:
Consistency; prior to this point, all other fields are accessed by strings and the id is accessed by this symbol. After this, all are strings.